average.js

```diff
@@ -1,6 +1,9 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const NaNumbers = numbers.filter(function (num) {
+    return !Number.isNaN(num);
+  });
+
+  return NaNumbers.reduce((p, c) => p + c, 0) / NaNumbers.length;
 }
 
-module.exports = {average};
+module.exports = { average };
```

Comment:
Try a new test: The tests in `test/average.test.js` express the functionality of the `average` function.
Consider this situation: In IoT applications, analog sensors will report occasional outliers. 'Outliers' are values so small or so large, they aren't expected to happen. We need new functionality to ignore these outliers.
First add a unit-test in `test/average.test.js` to ignore outliers. Name the test appropriately. Then implement it in the code.
You'll need thresholds to recognize outliers: Think of sensing battery-temperature values and determine reasonable values.

Reply:
Hey, @numocityadmin @sudeeprp I have added the outliers test. I used [this](https://mathworld.wolfram.com/Outlier.html) as the reference to find dynamic lower and upper bounds. I have tested using a couple of values and it is giving correct results for me. 

Please let me know what changes or features I need to add next 🤓